UIKitDZ/UIKitDZ/Choose Coffee screen/Ingridients screen/IngredientsViewController.swift

```diff
@@ -0,0 +1,76 @@
+// IngredientsViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Класс для отображение модельного окна с выбором ингридиентов
+final class IngredientsViewController: UIViewController {
+    // MARK: - Public Properties
+
+    /// Замыкание для передачи данных(в нашем случае обновленной цены)
+    var summaUp: ((String) -> Void)?
```

Comment:
```suggestion
    var summaUp: ((String) -> Void)?
```
нейминг заканчиваем на Handler у замыканий